sklearn/model_selection/_validation.py

```diff
@@ -106,19 +114,49 @@ def cross_val_score(estimator, X, y=None, groups=None, scoring=None, cv=None,
 
     Returns
     -------
-    scores : array of float, shape=(len(list(cv)),)
+    scores : array of float or dict of such arrays, array shape=(n_splits,)
         Array of scores of the estimator for each run of the cross validation.
 
+        For multi-metric evaluation, a dict of arrays containing the score
+        arrays for each scorer is returned.
+
     Examples
     --------
     >>> from sklearn import datasets, linear_model
     >>> from sklearn.model_selection import cross_val_score
+    >>> from sklearn.metrics.scorer import make_scorer
+    >>> from sklearn.metrics import confusion_matrix
+    >>> from sklearn.svm import LinearSVC
     >>> diabetes = datasets.load_diabetes()
     >>> X = diabetes.data[:150]
     >>> y = diabetes.target[:150]
     >>> lasso = linear_model.Lasso()
-    >>> print(cross_val_score(lasso, X, y))  # doctest: +ELLIPSIS
-    [ 0.33150734  0.08022311  0.03531764]
+    >>> print(cross_val_score(lasso, X, y))               # doctest: +ELLIPSIS
+    [ 0.33...  0.08...  0.03...]
+
+    >>> # Multi-metric evaluation using cross_val_score
+    >>> # (Please refer the ``scoring`` parameter doc for more information)
+    >>> scores = cross_val_score(lasso, X, y,
+    ...                          scoring=('r2', 'neg_mean_squared_error'))
+    >>> print(scores['neg_mean_squared_error'])           # doctest: +ELLIPSIS
+    [-3635.5... -3573.3... -6114.7...]
+    >>> print(scores['r2'])                               # doctest: +ELLIPSIS
+    [ 0.33...  0.08...  0.03...]
+
+    >>> # A sample toy binary classification dataset
```

Comment:
I would use this to the user guide to the section I just proposed. this example with the lambdas is a bit involved.